Dockerfile.compute-node

```diff
@@ -337,11 +338,12 @@ USER root
 
 FROM rust-extensions-build AS pg-jsonschema-pg-build
 
-RUN git clone --depth=1 --single-branch --branch neon_abi_v0.1.4 https://github.com/vadim2404/pg_jsonschema/ && \
+RUN git clone --single-branch https://github.com/supabase/pg_jsonschema && \
     cd pg_jsonschema && \
+    # there is no release tag yet, but we need it due to the superuser fix in the control file
+    git checkout caeab60 && \
```

Comment:
look at `pg_jwt` extension how are we downloading the archive with specific commit

Reply:
fixed